src/lib/storage/index/group_key/composite_group_key_index.cpp

```diff
@@ -46,17 +46,17 @@ CompositeGroupKeyIndex::CompositeGroupKeyIndex(
     Assert(static_cast<bool>(dict_segment), "CompositeGroupKeyIndex only works with dictionary segments.");
     Assert(dict_segment->compressed_vector_type(),
            "Expected DictionarySegment to use vector compression for attribute vector");
-    Assert(is_fixed_size_byte_aligned(*dict_segment->compressed_vector_type()),
-           "CompositeGroupKeyIndex only works with fixed-size byte-aligned compressed attribute vectors.");
+    Assert(is_fixed_width_integer(*dict_segment->compressed_vector_type()),
+           "CompositeGroupKeyIndex only works with Fixed-width integer compressed attribute vectors.");
     _indexed_segments.emplace_back(dict_segment);
   }
 
   // retrieve amount of memory consumed by each concatenated key
```

Comment:
Not yours: it's not amount. Retrieve memory consumption?

Reply:
meinetwegen :D

Reply:
"Amount" is for uncountable things. That should rather "Oh, ja wirklich, mea culpa; wie konnte ich nur?" than a "meinetwegen". :D

Reply:
Unless the blame shows that's my comment. Then, it's acceptable, but we should still change it.